### ISSUE TITLE
Latest Version Not working

### ISSUE THREAD

**Issue by mskian** [dyaa/ssl-checker#22]
Latest Version is Not Working

`index.js`

```js
var sslChecker = require("ssl-checker")
sslChecker("dyaa.me", { method: "GET", port: 443 }).then(console.info);
```

```js
/home/groot/Santhosh/nodejs/ssl-expiry-reminder/test/index.js:2
sslChecker("dyaa.me", { method: "GET", port: 443 }).then(console.info);
^

TypeError: sslChecker is not a function
    at Object.<anonymous> (/home/groot/Santhosh/nodejs/ssl-expiry-reminder/test/index.js:2:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

Node Version - `v10.16.3`

**Comment by dyaa** [comment]
Just noticed that this morning, A fix for this is coming soon, Thanks for reporting

**Comment by dyaa** [comment]
Fixed within [ssl-checker@2.0.3](https://github.com/dyaa/ssl-checker/releases/tag/v2.0.3)

Thanks again for reporting

**Comment by mskian** [comment]
@dyaa  Now Perfect :100:
